testing.rst

```diff
@@ -284,8 +284,9 @@ document::
 
         // Assert that the response is a redirect to /demo/contact
         $this->assertTrue(
-            $client->getResponse()->isRedirect('/demo/contact'),
-            'response is a redirect to /demo/contact'
+            $client->getResponse()->isRedirect('/demo/contact')
```

Comment:
To ease debugging, I learned to favor:
 ```php
$this->assertSame(Response::HTTP_FOUND, $client->getResponse()->getStatusCode());
$this->assertSame('/demo/contact', $client->getResponse()->headers->get('location'));
```
And since it's very verbose and repetitive I've got a ControllerTestTrait which provides assertions like:
```php
public function assertResponseStatusCode(Client $client, $code)
{
    $this->assertSame($code, $client->getResponse()->getStatusCode());
}

public function assertClientIsRedirectedTo(Client $client, $url, $permanent = false)
{
    $this->assertResponseStatusCode($client, $permanent ? Response::HTTP_MOVED_PERMANENTLY : Response::HTTP_FOUND);
    $this->assertSame($url, $client->getResponse()->headers->get('location'));
}
```
As said in another PR, we need to avoid failure messages like "true is not false" as much as possible to debug faster. Comparing values gives insight immediately.

Maybe it is worth to update the example to advise it, what do you think?